templates/strings/objc-m.stencil

```diff
@@ -0,0 +1,80 @@
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
+{% if tables.count > 0 %}
+
+#import "Localizable.h"
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wformat-security"
+
+@interface BundleToken : NSObject
+@end
+
+@implementation BundleToken
+@end
+
+NSString* tr(NSString* tableName, NSString* key, ...) {
```

Comment:
Can a free standing function like this produce issues (conflicts)? We may want to embed it in a type (interface) or something.

Reply:
Nitpick: have the `*` stick to the name, not the type:
`NSString *tr(NSString *tableName, NSString *key, ...) {`

Apply the same to the generated code.

Reply:
I'll make `tr()` a `static` function, it won't be accessible from outside the file.

I agree with the nitpick!

so, coming soon: `static tr(NSString *tableName, NSString *key, ...)`